po/fr.po

```diff
@@ -6057,6 +6126,41 @@ msgid_plural "%s have been imported"
 msgstr[0] "%s a été importé"
 msgstr[1] "%s ont été importés"
 
+#: ../src/develop/masks/brush.c:1049 ../src/develop/masks/brush.c:1134
+#, c-format
+msgid "hardness: %3.2f%%"
+msgstr "dureté : %3.2f%%"
+
+#: ../src/develop/masks/brush.c:1073 ../src/develop/masks/brush.c:1116
+#: ../src/develop/masks/brush.c:1123 ../src/develop/masks/circle.c:107
+#: ../src/develop/masks/circle.c:161 ../src/develop/masks/ellipse.c:448
+#: ../src/develop/masks/ellipse.c:535 ../src/develop/masks/path.c:1001
+#, c-format
+msgid "size: %3.2f%%"
+msgstr "taille : %3.2f%%"
+
+#: ../src/develop/masks/circle.c:94 ../src/develop/masks/circle.c:144
+#: ../src/develop/masks/ellipse.c:420 ../src/develop/masks/ellipse.c:505
+#: ../src/develop/masks/path.c:938 ../src/develop/masks/path.c:945
+#, c-format
+msgid "feather size: %3.2f%%"
+msgstr "taille d'adoucissement : %3.2f%%"
```

Comment:
I would use just "adoucissement : %3.2f%%"

This is a transient toast message, shorter will be easier to read and it seems clear that we are talking about size here.

Reply:
ok